src/copygb2/copygb2.f

```diff
@@ -1,183 +1,170 @@
-C>      areas of the map, at least where the merge field
-C>      is not bitmapped.  No merging is done if no merge
-C>      field is found.
-C>
-C>   -N namelist
-C>      Namelist file to override default output options.
-C>      The namelist must start with " &NLCOPYGB" and end with "/".
-C>      Namelist variables are
-C>        IDS(255)      Output decimal scaling by parameter
-C>        IBS(255)      Output binary scaling by parameter
-C>        NBS(255)      Output number of bits by parameter
-C>      
-C>   -v "uparms"
-C>      Parameter indicator(s) for the u-component of vectors.
-C>      A specific parameter is indicated by three numbers:
-C>      disc|cat|num.  Any parameter value in uparms is defined
-C>      as (65536*disc)+(256*cat)+num.
-C>      The parameter indicator for the v-component is assumed
-C>      to be one more than that of the u-component.
-C>      If the -v option is not specified, then the wind
-C>      components (parameters 0|2|2 = 514 and 0|2|3 = 515)
-C>      are the only fields assumed to be vector components
-C>      in the GRIB file.
-C>
-C>   -x
-C>      Turns off the use of an index file.  The index records
-C>      are then extracted from the GRIB file, which
-C>      will increase the time taken by copygb2.
-C>
-C>   -X
-C>      Turns on verbose printout.  This option is
-C>      incompatible with GRIB output to standard output.
-C>
-C> INPUT FILES:
-C>   - UNIT   11    INPUT GRIB FILE
-C>   - UNIT   14    MAP GRIB FILE
-C>   - UNIT   15    MERGE GRIB FILE
-C>   - UNIT   31    INPUT GRIB INDEX FILE
-C>   - UNIT   34    MAP GRIB INDEX FILE
-C>   - UNIT   35    MERGE GRIB INDEX FILE
-C>
-C> OUTPUT FILES:
-C>   - UNIT   51    OUTPUT GRIB FILE
-C>
-C> SUBPROGRAMS CALLED:
-C>   - IARGC
-C>   - GETARG
-C>   - ERRMSG
-C>   - EUSAGE
-C>   - ERREXIT
-C>   - FPARSEI
-C>   - FPARSER
-C>   - BAOPENR
-C>   - BAOPENWT
-C>   - BAOPENWA
-C>   - CPGB
-C>      
+!> @file
+!> @brief Copy all or part of one GRIB2 file to another GRIB2 file.
+!> @author Iredell @date 1998-10-22
+!
+!> The command copygb2 copies all or part of one GRIB2 file
+!> to another GRIB2 file, interpolating if necessary.  Unless
+!> otherwise directed (-x option), the GRIB2 index file is also used
+!> to speed the reading. The fields are interpolated to an output grid
+!> if specified (-g option). The interpolation type defaults to
+!> bilinear but may be specified directly (-i option).  The copying
+!> may be limited to specific fields (-k option). It may also be
+!> limited to a specified subgrid of the output grid or to a subrange
+!> of the input fields (-B and -b, -A, and -K options). Fields can be
+!> identified as scalars or vectors (-v option), which are interpolated
+!> differently.  The invalid data in the output field can be filled
+!> with mask values or merged with a merge field (-M and -m options).
+!> The output GRIB2 message can also be appended to a file (-a option).
+!> Some defaults can be overridden in a namelist file (-N option).
```

Comment:
Some periods have two spaces after them. Change to 1

Reply:
I have done this. It's going to take a few passes to get this documentation up to standard...